src/Files.App/App.xaml.cs

```diff
@@ -153,7 +153,13 @@ private IHost ConfigureHost()
 					.AddSingleton<DrivesViewModel>()
 					.AddSingleton<NetworkDrivesViewModel>()
 					.AddSingleton<OngoingTasksViewModel>()
-					.AddSingleton<AppearanceViewModel>()
+					// Settings ViewModels
```

Comment:
What benefit does it bring over just instantiating view models?

Reply:
we can use logger and services that are on that list if you add them into arguments of the constructor that injected by ioc. this is the very benefit.

Reply:
example: https://github.com/FluentHub/FluentHub/blob/main/src/FluentHub.App/ViewModels/Users/IssuesViewModel.cs#L16

Reply:
@d2dyno1

Reply:
It's weird, but I cannot mark this conversation as resolved...